lib/radar/meetup.rb

```diff
@@ -11,8 +12,14 @@ def next_events
       end
     end
 
+    # Optimistic group name parser for meetup.com.
+    # Will for example extract 'Git-Aficionados'
+    # from http://www.meetup.com/Git-Aficionados/events/155514542/
+    # As @radar_setting.url is user input we shouldn't be too naive.
     def group_urlname
-      @radar_setting.url.match(/http(?:s?):\/\/www.meetup.com\/([^\/]*)/)[1]
+      url = Addressable::URI.heuristic_parse(@radar_setting.url)
+      parts = url.path.split('/')
+      parts.second || ''
```

Comment:
Even though this is correct, I found it to be confusing as I wasn't aware that the path would have a leading slash.